pkg/model/stack_expander_test.go

```diff
@@ -26,6 +26,8 @@ services:
             - $CUSTOM_ENV
             - CUSTOM2_ENV=$CUSTOM_ENV
             - EMPTY
+            - ${ENV:-dev}
+            - ${ENV}
```

Comment:
Can we add `$ENV`?

Reply:
is that the same case as `$CUSTOM_ENV` ¿? first of list?